pcweb/pages/docs/component.py

```diff
@@ -582,7 +583,10 @@ def same_trigger(t1, t2):
     return args1 == args2
 
 
-def generate_event_triggers(comp):
+def generate_event_triggers(comp, src):
+    prop_name_to_description = {
+        prop.name: prop.description for prop in src.get_props()
```

Comment:
I'd expect a `if prop.name.startswith("on_")` to mirror the l506 ? 
Am I missing something obvious ?